dashboard/app/helpers/aichat_openai_helper.rb

```diff
@@ -3,190 +3,56 @@
 class OpenaiUserInputResponseTimeout < StandardError; end
 
 # Prepares the input (user/level system prompt, context, existing chat history)
-# from AI Chat lab to be sent to the OpenAI API, and sends the request to the API.
-#
-# This module is structured very similarly to the AichatSagemakerHelper module,
-# which manages AI Chat lab's interaction with models that use AWS Sagemaker.
-module AichatOpenaiHelper
-  API_KEY = CDO.openai_student_learning_api_key
-  MODEL = SharedConstants::AICHAT_MODEL_VERSION
+# from AI Chat lab to be sent to the AI API, and then sends the request to the API.
 
+module AichatOpenaiHelper
```

Comment:
Could we kill this helper entirely? Seems like most of what it does (unwrapping `aichat_customizations`) could be done in the new client or the active job code.